src/moonstone-samples/lib/DataGridListSample.js

```diff
@@ -110,12 +147,72 @@ module.exports = kind({
 		this.modelIndex = idx;
 		return records;
 	},
+	generateDataGridList: function(itemType) {
+		if (this.$.gridList) {
+			this.$.gridList.destroy();
+		}
+
+		var moreProps = {};
```

Comment:
Might be simpler to clone `dataListDefaults` here and then modify that copy in the conditional block below. Results in fewer objects created as well and while that's not critical for a sample, it does demonstrate better practices for anyone that might base their production code off of the sample.